src/collections/_documentation/cli/releases.md

```diff
@@ -72,6 +72,14 @@ If you also want to set a previous commit instead of letting the server use the
 sentry-cli releases set-commits "$VERSION" --commit "my-repo@from..to"
 ```
 
+### Alternatively: Without a Repository Integration
+You can still use the `--auto` flag and the cli will automatically use the git tree of your local repo, and associate commits between the previous release's commit and the current head commit with the release. If this is the first release, Sentry will use the latest 20 commits. This behavior is configurable with the `--initial-depth` flag. 
```

Comment:
```suggestion
You can still use the `--auto` flag, and the CLI will automatically use the git tree of your local repo, and associate commits between the previous release's commit and the current head commit with the release. If this is the first release, Sentry will use the latest 20 commits. This behavior is configurable with the `--initial-depth` flag. 
```

Reply:
@MimiDumpling Sorry I missed this suggestion and merged. I'm going to make a new PR for this.

Reply:
PR for this is https://github.com/getsentry/sentry-docs/pull/1832